.github/workflows/github-build.yml

```
@@ -31,6 +31,11 @@ jobs:
           submodules: "recursive"
           fetch-depth: 0
 
+      # continuous tag confuses git describe --tags, happens here specifically due to us fetching the entirety of the repository
+      - name: Delete continuous tag
+        run: |
+          git tag | grep '^continuous$' && git tag -d continuous
```

```suggestion
          git tag -d continuous 2> /dev/null
```

Save some cpu cylces?

---

It'll fail if the tag doesn't exist with return code 1, causing the step to fail. You'd have to `|| true` then. It won't really speed it up either, I guess. Also, allowing the output to end up in the log is handy for debugging.

---

You could do the following:

```suggestion
          git tag -d continuous || true
```